TShockAPI/Bouncer.cs

```diff
@@ -2491,6 +2491,11 @@ internal void OnPlaceItemFrame(object sender, GetDataHandlers.PlaceItemFrameEven
 
 			if (!args.Player.HasBuildPermission(args.X, args.Y))
 			{
+				int num = Item.NewItem(new EntitySource_DebugCommand(), (args.X * 16) + 8, (args.Y * 16) + 8, args.Player.TPlayer.width, args.Player.TPlayer.height, args.ItemID, args.Stack, noBroadcast: true, args.Prefix, noGrabDelay: true);
+				Main.item[num].playerIndexTheItemIsReservedFor = args.Player.Index;
+				NetMessage.SendData((int)PacketTypes.ItemDrop, -1, -1, NetworkText.Empty, num, 1f);
+				NetMessage.SendData((int)PacketTypes.ItemOwner, -1, -1, NetworkText.Empty, num);
```

Comment:
```suggestion
				NetMessage.SendData((int)PacketTypes.ItemDrop, args.Player.Index, -1, NetworkText.Empty, num, 1f);
				NetMessage.SendData((int)PacketTypes.ItemOwner, args.Player.Index, -1, NetworkText.Empty, num);
```
I'm suggesting a change so that the item drop is not visible to anyone else

Reply:
this changes a bit when a player is running and a item doesn't reach the player it will update the item which other players can pick it up but its a good improvement 👍